presto-iceberg/src/main/java/io/prestosql/plugin/iceberg/IcebergMetadataFactory.java

```diff
@@ -37,23 +38,24 @@ public IcebergMetadataFactory(
             TypeManager typeManager,
             JsonCodec<CommitTaskData> commitTaskDataJsonCodec)
     {
-        this(metastore, hdfsEnvironment, typeManager, commitTaskDataJsonCodec);
+        this(metastore, hdfsEnvironment, typeManager, commitTaskDataJsonCodec, config.isUniqueTableLocation());
     }
 
     public IcebergMetadataFactory(
             HiveMetastore metastore,
             HdfsEnvironment hdfsEnvironment,
             TypeManager typeManager,
-            JsonCodec<CommitTaskData> commitTaskCodec)
+            JsonCodec<CommitTaskData> commitTaskCodec, boolean useUniqueTableLocation)
```

Comment:
Wrap the argument, since all other arguments are wrapped

Reply:
Not sure that I understand comment correct, but moved ` boolean useUniqueTableLocation` to the next line